frontend/tests/components/opportunity/OpportunityDescription.test.tsx

```diff
@@ -87,7 +87,9 @@ describe("OpportunityDescription", () => {
     expect(additionalInfoHeading).toBeInTheDocument();
 
     const sanitizedEligibilityDescription = DOMPurify.sanitize(
-      mockOpportunityData.summary.applicant_eligibility_description,
+      mockOpportunityData.summary.applicant_eligibility_description
```

Comment:
I know these tests are not the point of this change, so feel free to ignore at this point but after thinking about this some more, I think it would be best to just remove the calls to `sanitize` on lines 61 and 89 and leave the tests the same otherwise. The tests still pass, as the values we're sending in in the mock do not change when run through the sanitize function, as it is being mocked to be just an identity.